packages/ui-components/src/components/breadcrumb-list/breadcrumb-list.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Host, h, Prop, Element } from '@stencil/core';
+
+@Component({
+	tag: 'kv-breadcrumb-list',
+	styleUrl: 'breadcrumb-list.scss',
+	shadow: true
+})
+export class KvBreadcrumbList {
+	/** (optional) The separator to use between breadcrumb items */
+	@Prop() separator?: string = '/';
+
+	/** The Host's element reference */
+	@Element() el: HTMLKvBreadcrumbListElement;
+
+	private breadcrumbItems: HTMLKvBreadcrumbItemElement[];
+
+	componentDidRender() {
+		this.breadcrumbItems = Array.from(this.el.querySelectorAll('kv-breadcrumb-item'));
+		this.breadcrumbItems.forEach(item => (item.separator = this.separator));
+	}
```

Comment:
As I mentioned early, it is my opinion that this should only be on the `breadcrumb.tsx` component.